static/src/javascripts/projects/commercial/modules/hosted/youtube.js

```diff
@@ -90,7 +90,7 @@ export const initHostedYoutube = (el: HTMLElement): void => {
                         );
                     }, 1000);
                 } else {
-                    clearTimeout(playTimer);
+                    window.clearInterval(playTimer);
```

Comment:
As before, we were previously passing `playTimer` to `clearTimeout` however `playTimer` is an `IntervalID` so we should be using `clearInterval`.